config/deploy.js

```diff
@@ -2,28 +2,38 @@
 'use strict';
 
 module.exports = function (deployTarget) {
+	const distDir = 'docs-dev';
+
 	let ENV = {
 		build: {
-			outputPath: 'docs-dev',
+			outputPath: distDir,
 		},
+		plugins: ['build']
 		// include other plugin configuration that applies to all deploy targets here
 	};
 
 	if (deployTarget.startsWith('dev-')) {
 		ENV.build.environment = 'devbox';
+		ENV.plugins.push('sftp');
 
 		ENV.sftp = {
 			host: deployTarget,
-			distDir: 'docs-dev',
+			distDir: distDir,
 			remoteDir: '/var/www/design-system',
 			remoteUser: deployTarget.replace('dev-', ''),
 			agent: process.env.SSH_AUTH_SOCK
 		};
 	}
 
-	if (deployTarget === 'production') {
+	if (deployTarget === 'github') {
 		ENV.build.environment = 'production';
-		// configure other plugins for production deploy target here
+		ENV.plugins.push('git');
+
+		ENV.git = {
+			repo: 'git@github.com:wikia/design-system.git',
+			branch: 'gh-pages',
```

Comment:
Instead of `gh-pages` branch you could use `docs/` dir instead.

Reply:
yep, that is how it is right now. 
but that means you have to commit alraedy built reference page

i think workflow
where you do update, commit, merge to master
and then release is more common

thoughts?

Reply:
thanks for reviewing!

Reply:
We have the same flow in the react mirror, but we're not using `gh-pages` - it seems simpler that way, as this compiled HTML is always accessible and can be executed from locally (`file://`).